addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -403,27 +407,37 @@
     </t>
 </template>
 
-<!-- Edit Options -->
+<!-- Button to configure Tickets -->
+<template id="registration_configure_tickets_button" name="Registration Configure Ticket Button">
+    <div class="o_not_editable ml-2" role="link">
+        <a t-attf-href="/web#id=#{event.id}&amp;view_type=form&amp;model=event.event">
+            <i class="fa fa-gear mr-1" role="img" aria-label="Configure" title="Configure event tickets"/>
+            <em>Configure Tickets</em>
+        </a>
+    </div>
+</template>
+
+<!-- This template is called whenever the "Fold Ticket Details" option is checked on the website (Customize > Fold Tickets Details) -->
 <template id="fold_register_details" inherit_id="website_event.registration_template" active="False" customize_show="True" name="Fold ticket details">
     <xpath expr="//div[@id='o_wevent_tickets']" position="attributes">
         <attribute name="data-folded-by-default">1</attribute>
     </xpath>
     <xpath expr="//div[@id='registered']" position="attributes">
         <attribute name="class">ml-auto pr-3</attribute>
     </xpath>
-    <xpath expr="//div[@id='price-range']" position="attributes">
-        <attribute name="class">ml-0 pr-3</attribute>
-    </xpath>
     <xpath expr="//div[@id='o_wevent_tickets_collapse']" position="attributes">
         <attribute name="class">collapse</attribute>
     </xpath>
+    <xpath expr="//span[@id='tickets-title']" position="attributes">
+        <attribute name="class">py-2 o_wevent_registration_title text-left d-none</attribute>
+    </xpath>
```

Comment:
I know the old code had ID based selectors, but I'm not a fan of those, not nicely prefixed and a bit messy.

Maybe you could do that instead:
```xml
<xpath expr="//span[hasclass('o_wevent_registration_title')]" position="attributes">
    <attribute name="class" add="d-none" separator=" "/>
</xpath>
```

Reply:
I don't mind using 'hasclass('o_wevent_registration_title')' in this situation, but I'll still have to use the id for the price-range bellow ? (see in the other comment: can't just get rid of it)